crystal_toolkit.egg-info/PKG-INFO

```diff
@@ -0,0 +1,18 @@
+Metadata-Version: 2.1
```

Comment:
The files in `crystal_toolkit.egg-info/*` should be `.gitignored`.

Reply:
Same with `dist/crystal_toolkit-2021.4.29-py3.9.egg` and anything else in `dist/*`.